gradle/documentation/render-javadoc.gradle

```diff
@@ -151,8 +151,25 @@ configure(project(":solr:test-framework")) {
         "org.apache.solr.analysis",
         "org.apache.solr.cloud",
         "org.apache.solr.core",
+        "org.apache.solr.handler",
         "org.apache.solr.handler.component",
         "org.apache.solr.update.processor",
+        "org.apache.solr.util",
+        "org.apache.solr.search.similarities",
+        "org.apache.solr.search.function",
+        "org.apache.solr.search.facet",
+        "org.apache.solr.schema"
```

Comment:
Is this needed?

Reply:
Ok so I looked into this. A handful of classes were moved from solr core into solr test-framework due to limitations in gradle not being able to share test classes. The move ended up in a few more split packages - which causes issues in the render-javadoc. So these exclusions are due to the new split packages. I didn't see a simple way to address this related to this change. 

I think test-framework should probably look at how it does packages (maybe need to at `.test.` or something to the package name) to avoid split packages.